backend/src/main/java/gov/cdc/usds/simplereport/service/ResultsUploaderDeviceValidationService.java

```diff
@@ -20,6 +25,54 @@
 @RequiredArgsConstructor
 public class ResultsUploaderDeviceValidationService {
   private final DeviceTypeRepository deviceTypeRepository;
+  private final SpecimenTypeRepository specimenTypeRepository;
+
+  private static final Map<String, String> specimenSNOMEDMap = new HashMap<>();
```

Comment:
can you replace this with a Map.of?

Reply:
So, `Map.of` will only accept up to ten key-value pairs - any more, and we'll get a compilation error (sounds a little goofy, but [the explanation](https://stackoverflow.com/questions/46601959/what-is-the-use-of-map-ofentries-instead-of-map-of#comment101525046_46604210) makes sense).

It looks like the third, and probably intended, way is to use `Map.ofEntries`, which accepts an arbitrary number of `Entry` key-value objects:

```
Map<String, String> items = Map.ofEntries(
    Map.entry("key-1", "value-1"),
    Map.entry("key-2", "value-2"),
    [...],
    Map.entry("key-10", "value-10"),
    Map.entry("key-11", "value-11"));
```

If you or any other review prefers this, I'll make the change!

Reply:
hmm, Does the `Map.ofEntries()` syntax eliminate the static block? It better for the declaration and assignment to be at the same time for static maps

Reply:
It would - I'll push the change 👏